UnityGLTF/Assets/UnityGLTF/Runtime/Plugins/GLTFSerialization/Extensions/KHR_draco_mesh_compression.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Collections.Generic;
+using GLTF.Extensions;
+using Newtonsoft.Json.Linq;
+
+namespace GLTF.Schema
+{
+	[Serializable]
+	public class KHR_draco_mesh_compression : IExtension
+	{
+		public Dictionary<string, int> attributes = new Dictionary<string, int>();
+		public BufferViewId bufferView;
+
+		public JProperty Serialize()
+		{
+			throw new NotSupportedException("KHR_draco_mesh_compression serialization is not supported yet.");
+		}
+
+		public IExtension Clone(GLTFRoot root)
+		{
+			return new KHR_draco_mesh_compression()
+			{
+				attributes = new Dictionary<string, int>(attributes),
+			};
+		}
+	}
+	public class KHR_draco_mesh_compression_Factory : ExtensionFactory
```

Comment:
Nit: make sure to have proper formatting / line breaks

Reply:
825268c674f3de83335c782103f09cc3f77678f1